scripts/build.sh

```diff
@@ -1,73 +0,0 @@
-#!/bin/bash
```

Comment:
`setup` and `build` scripts need to remain so the switch to igniter is transparent to users